pif_ingestor/core.py

```diff
@@ -51,16 +51,25 @@ def main():
     ingest_manager = IngesterManager()
 
     all_files = []
+    exceptions = []
     if args.recursive:
         for root, dirs, files in walk(args.path):
             try:
                 new = _handle_pif(root, args.format, args.converter_arguments, enrichment_args, ingest_manager)
                 all_files.extend(new)
-            except:
-                pass
+            except Exception as err:
+                exceptions.append(err)
```

Comment:
Can you pair the `err` with the `root` so you see like `"/path/to/failed/file.dat: JsonParseError"`